elodie/filesystem.py

```diff
@@ -80,6 +80,20 @@ def delete_directory_if_empty(self, directory_path):
 
         return False
 
+    def delete_file(self, file_path):
```

Comment:
We should add tests for this including for when `delete_duplicates` is `True` and `False`.